lib/app.rb

```diff
@@ -143,18 +140,14 @@ def option_selector(option)
     end
   end
 
-  def interface_menu
-    puts 'Please choose an option by entering a number:'
-    puts '1 - List all books'
-    puts '2 - List all people'
-    puts '3 - Create a person'
-    puts '4 - Create a book'
-    puts '5 - Create a rental'
-    puts '6 - List all rentals for a given person id'
-    puts '7 - Exit'
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/234861901-054432b4-fda8-4678-b8d4-b95749f6f729.png)

- [ ] After creating a new person, books, and rentals when I tried to exit your app then I saw this error which you can see in the image above :arrow_up: and your data is not being loaded from or saved in the json files :fearful: Kindly fix this error to make your app work as expected :+1:

Reply:
@HAMMAS-SALEEM this is due to not running the app from the lib/ folder